src/lib.rs

```diff
@@ -103,13 +105,15 @@ pub fn parse_module(
 
 /// Parse an individual module from an AST, returning the module.
 pub fn parse_module_from_ast(
+  graph_kind: GraphKind,
   specifier: &ModuleSpecifier,
   maybe_headers: Option<&HashMap<String, String>>,
   parsed_source: &deno_ast::ParsedSource,
   maybe_resolver: Option<&dyn Resolver>,
   maybe_npm_resolver: Option<&dyn NpmResolver>,
```

Comment:
So many parameters. Something to clean up in the future.

Reply:
Agreed, we should add `ParseModuleConfig` struct for this